keyboards/sol/keymaps/xulkal/keymap.c

```diff
@@ -92,15 +92,29 @@ static void render_logo(void) {
   oled_write_P(sol_logo, false);
 }
 
+extern rgb_config_t rgb_matrix_config;
+
 static void render_status(void) {
   // Render to mode icon
   static const char PROGMEM mode_logo[2][4] = {
```

Comment:
```suggestion
  static const char PROGMEM mode_logo[][] = {
```

Travis didn't pick up on this, for some reason it built all the default keymaps.

Reply:
fixed

Reply:
@fauxpark it's because userspace files were touched.   

Travis doesn't know how to handle that, and just assumes "test everything".